jwst/dq_init/tests/test_dq_init.py

```diff
@@ -0,0 +1,340 @@
+import pytest
+from astropy.io import fits
+from jwst.dq_init import DQInitStep
+from jwst.dq_init.dq_initialization import do_dqinit
+from jwst.datamodels import MIRIRampModel, MaskModel
+from jwst.datamodels import GuiderRawModel, RampModel
+from jwst.datamodels import ImageModel, dqflags
+import numpy as np
+import numpy.testing as nptest
+
+
+# Set parameters for multiple runs of data
+args = "xstart, ystart, xsize, ysize, nints, ngroups, instrument, exp_type"
+test_data = [(1, 1, 2304, 2048, 2, 2, 'FGS', 'FGS_ID-STACK'),
+             (1, 1, 2048, 2048, 2, 2, 'FGS', 'FGS_ID-IMAGE'),
+             (1, 1, 2048, 2048, 2, 2, 'NIRCAM', 'NRC_IMAGE'),
+             (1, 1, 1032, 1024, 1, 5, 'MIRI', 'MIR_IMAGE')]
+ids = ["GuiderRawModel-Stack", "GuiderRawModel-Image", "RampModel", "MIRIRampModel"]
+
+
+@pytest.mark.parametrize(args, test_data, ids=ids)
+def test_dq_im(xstart, ystart, xsize, ysize, nints, ngroups, instrument, exp_type):
+    # Check that PIXELDQ is initialized with the information from the reference file.
```

Comment:
Could you convert these to docstrings for the tests.  Something like
```
"""Check that flagged value in the reference file flags the DQ array in science image"""
```
or similar.  A one-liner is good.  But the triple-quotes are preferred `#`.

Reply:
Do you want just the description comments as triple quotes, or all comments?

Reply:
No comments are fin as is.  Just the docstring after `def foo()`.  These are special in python and are used to generate documentation.  Just for consistency's sake.  =)